sasjs-tests/src/testSuites/Basic.ts

```diff
@@ -151,6 +151,9 @@ export const basicTests = (
       description:
         'Should complete successful request with extra attributes present in response',
       test: async () => {
+        if (adapter.getSasjsConfig().serverType !== 'SASVIYA')
```

Comment:
Will this cause any issues when we add our third servertype (SASJS Server) ?

Reply:
Indeed it will

Reply:
No, because it will run only on viya, even tho check is counterintuitive